FreeRTOS/FreeRTOS-Partner-Supported-Demos
# Add Cortex-M35P fast-model blinky example

Add Cortex-M35P fast-model blinky example

Description
-----------
The blinky example supports two configurations, blinky demo and full demo. The blinky demo prints a message to the console when a receiver task receives an expected message from the sender task. The full demo implements soak testing of FreeRTOS kernel on Cortex-M35P fast-model.

Test Steps
-----------
The blinky and full example validated on Armv8-M Cortex-M35P fast-model.

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
